rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.cpp

```diff
@@ -76,6 +77,88 @@ SqliteWrapper::~SqliteWrapper()
   }
 }
 
+void SqliteWrapper::apply_pragma_settings(
+  std::vector<std::string> & pragmas,
+  rosbag2_storage::storage_interfaces::IOFlag io_flag)
+{
+  // sqlite pragmas are assigned with either '= value' or '(value)' syntax, depending on pragma
+  const std::string pragma_assign = "=";
+  const std::string pragma_bracket = "(";
+
+  // Apply default pragmas if not overridden by user setting
+  {
+    // Used to check whether db is readable
+    const std::string schema = "schema_version";
+
+    typedef std::map<std::string, std::string> pragmas_map;
```

Comment:
do these pragmas have to ordered alphabetically? Or could we use a `unordered_map` here?

Reply:
absolute nitpick: I like to suffix these sorts of typedef's with `_t`, e.g. `pragmas_map_t`